app/processors/casino/two_factor_authenticator_registrator_processor.rb

```diff
@@ -1,23 +1,15 @@
 require 'rotp'
 
 # The TwoFactorAuthenticatorRegistrator processor can be used as the first step to register a new two-factor authenticator.
-# It is inactive until activated using TwoFactorAuthenticatorActivator.
+# It will create a secret that will be used by TwoFactorAuthenticatorAcceptor to verify the generated otp
 #
-# This feature is not described in the CAS specification so it's completly optional
-# to implement this on the web application side.
 class CASino::TwoFactorAuthenticatorRegistratorProcessor < CASino::Processor
   include CASino::ProcessorConcern::TicketGrantingTickets
 
-  # This method will call `#user_not_logged_in` or `#two_factor_authenticator_registered(two_factor_authenticator)` on the listener.
-  # @param [Hash] cookies cookies delivered by the client
-  # @param [String] user_agent user-agent delivered by the client
-  def process(cookies = nil, user_agent = nil)
-    cookies ||= {}
-    tgt = find_valid_ticket_granting_ticket(cookies[:tgt], user_agent)
-    if tgt.nil?
-      @listener.user_not_logged_in
-    else
-      two_factor_authenticator = tgt.user.two_factor_authenticators.create! secret: ROTP::Base32.random_base32
+  def process(user)
+    person = Person.find(user.extra_attributes[:person_id])
```

Comment:
same here, do we use the one of our clients gems here? 
btw if you have to use it then could you use the konvenit_client gem because @anikolskiy is removing the konvenit-activeresouce. `Reaktor::Person.find`

Reply:
here also Person is fetched from Reaktor

Reply:
yes it is a real database call
as I understand that we only have casino gem in Reaktor so it should be usable in the context of Reakor because it handles the login process.
while testing it worked well when I start login from Reaktor or even from any other app